application/templates/application/application-form.html

```diff
@@ -111,6 +114,7 @@ <h5 class="p-2">Educational</h5>
         <div class="rounded bg-secondary" style="margin-bottom:10px;margin-top:15px;">
             <h5 class="p-2">Selected Schools and Programs</h5>
         </div>
+        <p style="color:red; ">Schools will be listed from your favorites.</p>
```

Comment:
Rephrasing this to "Only your favorited schools will be available for selection." might be better.

Reply:
sure